src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/experimental_feature.yml

```
@@ -0,0 +1,15 @@
+admin_experimental_feature_index:
+    path: /
+    methods: GET
```

```suggestion
    methods: [GET, POST]
```
Or keep the second route, by it's look a little bit overkill

---

Fixed in https://github.com/PrestaShop/PrestaShop/pull/23703/commits/a95d4af99ad89a142b320d0011ddb9db2e5270cf